app/views/transport.scala.html

```diff
@@ -17,54 +17,121 @@
 @import controllers.routes
 @import forms.Transport
 @import forms.Transport.ModesOfTransport._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
 @import views.Title
-@import views.components.fields.RadioOption
+@import components.gds.{errorSummary, gds_main_template, pageTitle, sectionHeader}
 
-@this(main_template: views.html.main_template)
 
-@(form: Form[Transport])(implicit request: Request[_], messages: Messages)
+@this(  govukLayout: gds_main_template,
+        govukButton: GovukButton,
+        govukRadios: GovukRadios,
+        govukInput: GovukInput,
+        pageTitle: pageTitle,
+        errorSummary: errorSummary,
+        sectionHeader: sectionHeader,
+        formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF)
 
-@radio(mode: String) = { RadioOption(mode, mode, Transport.messageKey(mode)) }
+@(form: Form[Transport], consignmentReference: Option[String])(implicit request: Request[_], messages: Messages)
```

Comment:
Similar, should consignmentRef always be present here?

Reply:
In practice I think it will always be there.

Reply:
Do you think it should be changed @ed0906?   Ultimately something has to cope with the possibility that the value might not be there?

Reply:
We could change it in the controller to `consignmentReferences.getOrElse(ReturnToStartException)`

Reply:
I forgot this project has that magic exception.    I'll make the changes

Reply:
I think all the UIs have some sort of ReturnToStart exception now

Reply:
You can do it in the view rather than the controller if that's easier. It probably fits more naturally in the controller though